debug/programs/entry.S

```diff
@@ -204,6 +204,18 @@ trap_entry:
 loop_forever:
   j loop_forever
 
+  .align 2
+precease:
+    // Loop a while so that OpenOCD might have confirmed the resume before the
+    // hart becomes unavailable.
+    li t1, 100000
+1:
+    addi t1, t1, -1
+    bnez t1, 1b
+cease:
+  .word 0x30500073  // cease
```

Comment:
So I guess Spike supports the CEASE instruction, that's good

Reply:
It doesn't; it'll raise an illegal-instruction exception.

Reply:
Right. I've been running this against an FPGA image that does support the cease instruction.

Reply:
ah, yeah, got it, thanks!